File: Learnify.Domain/Entities/Enrollment.cs
namespace Learnify.Domain.Entities
{
    public class Enrollment : BaseEntity
    {
        public Guid StudentId { get; set; }
        public User? Student { get; set; }
        public Guid CourseId { get; set; }
        public Course? Course { get; set; }
        public float Progress { get; set; }
        public DateTime EnrolledAt { get; set; } = DateTime.UtcNow;
    }
}

File: Learnify.UI/src/models/entities/Enrollment.js
export class Enrollment {
    constructor(
        id,
        studentId,
        student = null,
        courseId,
        course = null,
        progress,
        enrolledAt = new Date()
    ) {
        this.id = id;
        this.studentId = studentId;
        this.student = student;
        this.courseId = courseId;
        this.course = course;
        this.progress = progress;
        this.enrolledAt = enrolledAt;
        
    }
}